nixos/modules/services/scheduling/scx.nix

```diff
@@ -0,0 +1,114 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}:
+let
+  cfg = config.services.scx;
+in
+{
+  options.services.scx = {
+    enable = lib.mkEnableOption null // {
+      description = ''
+        Whether to enable SCX service, a scheduler daemon with a wide variety of
+        scheduling algorithms to improve system performance.
```

Comment:
Again, claims of "improve system performance" are likely unfounded in the general case.

Reply:
Will remove it.